ops/ansible/roles/postmaster_deploy/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+
+- name: Install main list of packages
+  apt: pkg={{item}} state=installed update_cache=true
+  sudo: yes
+  with_items:
+    - git
+    - python
+    - python-pip
+    - python-dev
+    - python-virtualenv
+    - libldap2-dev
+    - libssl-dev
+    - libsasl2-dev
+    - libffi-dev
+    - apache2
+    - libapache2-mod-wsgi
+    - libmysqlclient-dev
+  when: provision_type == "dev"
```

Comment:
Exactly. You can't run dpkg inside of dpkg. So that statement is for dev only when ansible is called from Vagrant. 

https://github.com/StackFocus/PostMaster/blob/ansible_in_my_pansible/build_release.sh#L3